### PR TITLE
Update SC interface. Add ABI

### PR DESCRIPTION
**Pull request by BigRoma**
In turnOnWallet function don't need serial number and encrypted common secret changed to common secret

### CODE REVIEW

File: SecurityCardManagement/README.md [hunk 1]
```diff
@@ -36,10 +36,9 @@ Before forwarding the request to a security card, a browser must request a pin c
 arguments: 
 
 	answerId: uint32 - function id of result callback	
-	sn	: uint192 - card serial number
```
Comment (sertony) [inline SecurityCardManagement/README.md:39]: Why removing it? You could use this info to bind the keys data to the card ID

File: SecurityCardManagement/README.md [hunk 2]
```diff
@@ -36,10 +36,9 @@ Before forwarding the request to a security card, a browser must request a pin c
...
 	p1      : uint128 - authentication password
 	iv      : uint16 - vector for AES128 CBC initialization for encrypting P1
-	ecs     : uint32 - encrypted common secret
+	cs     : uint32 - common secret
```
Comment (sertony) [inline SecurityCardManagement/README.md]: Why is it not encrypted anymore? I believe this was made on purpose in order not to pass the plain unencrypted data via the interface

Comment (mikhailUshakoff) [reply SecurityCardManagement/README.md]: yep, just common secret here

File: SecurityCardManagement/README.md (hunk 1 above)
Comment (BigRoma) [reply SecurityCardManagement/README.md:39]: We can restore this, if really need it.

**Review by Keshoid — APPROVED** [review]

**Review by mikhailUshakoff — APPROVED** [review]
LGTM

**Review by sertony — APPROVED** [review]

**Pull request merged**